WcaOnRails/spec/lib/middlewares/warden_user_logger_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'middlewares/warden_user_logger'
+require 'ostruct'
+
+describe Middlewares::WardenUserLogger do
+  describe "call" do
+    let(:app) { double(:app) }
+    let(:logger) { double(:logger) }
+    subject { described_class.new(app, logger) }
+
+    before do
```

Comment:
Total nitpick, but it took me a while to figure this out...

Just checking, this looks like it's the same as `before :example`, which is the same as a `before :each`? (https://github.com/rspec/rspec-core/blob/7477810fc65718a69753f04d4a99c3acf2371acb/lib/rspec/core/hooks.rb#L13-L15)

We're using `before :each` everywhere else, so maybe change this to `before :each` for consistency?